lib/layout/box_layout.dart

```diff
@@ -154,7 +164,9 @@ class BoxLayoutState extends WidgetState<BoxLayout> with TemplatedWidgetState {
       children = updatedChildren;
     }
 
-    Widget boxWidget = _buildBoxWidget(children);
+    Widget boxWidget = templatedChildren != null && widget is ListView
+        ? _buildListView(children)
```

Comment:
if else is not ideal as ListView is very different than BoxWidget. Can you extract the barebone (didChangeDependencies(), dispose()) into a base class say "WidgetListState", then have both BoxLayoutState and a new ListViewState extend it?

Reply:
that way ListViewState is clean, and knows nothing about the Column/Row/...